scripts/README.md

```diff
@@ -1,67 +1,84 @@
-Contents of PEcAn scripts/ folder
----------------------------------
+# PEcAn Scripts Directory
```

Comment:
These look like the same changes as #3494 -- should they be removed from this PR?